src/Shift-ClassInstaller-Tests/ShClassInstallerTest.class.st

```diff
@@ -288,6 +287,30 @@ ShClassInstallerTest >> testDuplicatedInstanceVariableInSuperclassMetaclass [
 	self deny: ((superClass class slots collect: [:e | e name]) includes: 'aSlot')
 ]
 
+{ #category : #tests }
+ShClassInstallerTest >> testInstallInSpecificEnvironment [
+
+	| newEnvironment |
+	newEnvironment := SystemDictionary new.
+	newClass := ShiftClassInstaller new
+		            installingEnvironment: (ShSmalltalkGlobalsEnvironment new
+				             environment: newEnvironment;
+				             yourself);
+		            make: [ :builder |
+			            builder
+				            name: #SHClassForTest;
+				            superclass: Object;
+				            category: self generatedClassesPackageName ].
+
+	self class environment at: #SHClassForTest ifPresent: [ self fail: 'The default environment should not have the class installed.' ].
+	newEnvironment at: #SHClassForTest ifAbsent: [ self fail: 'The default environment should not have the class installed.' ].
+	
+	self skip. 
```

Comment:
Expect the failure instead of skip, so if is fixed on day it will be easier to remember to update the test.

Reply:
I'm currently working on RPackage and I hope I'll be able to fix this before Pharo 12 release. So I added a flag for me to find this test in the next few months